src/index.jsx

```diff
@@ -21,7 +21,7 @@ class Checkbox extends Component {
   constructor(props) {
     super(props);
 
-    const checked = 'checked' in props ? props.checked : props.defaultChecked;
+    const checked = 'defaultChecked' in props ? props.defaultChecked : 'checked' in props ? props.checked : false;
```

Comment:
Can the double ternary operator be avoided?

Reply:
It is wrong to read defaultChecked before checked, checked should be top priority.

Reply:
I need to look at the code again🧎‍♂️。I will reopen this pr after I fix it。